### ISSUE TITLE
Don't generate alt="" if no alt text was given

### ISSUE THREAD

**Issue by thibaultmol** [TryGhost/Ghost#22251]
### Issue Summary

Currently there seems to be a bug in Chrome Devtools which makes is so that alt="" is detected as "all good, we have alt text" which obv isn't true because no alt text was provided.
But it seems like ghost is creating alt="" on images when the Ghost publisher didn't provide any alt text. Ghost probably shouldn't putting the alt="" there is that case and only add the tag if there was indeed alt text provided.
While it's a lighthouse bug, various browsers might also not have checks for detecting an empty alt text string and that could confuse screen readers potentially. It's given a false sense of accessibility 

### Steps to Reproduce

Run a Ghost post through Lighthouse that has images without alt text.
https://pagespeed.web.dev/analysis/https-thelibre-news-would-you-pay-for-free-software/81gt39l14p?form_factor=mobile

that's an example of a page that (at the time of making this post) doesn't have alt text on some of the images.

### Ghost Version

n/a

### Node.js Version

n/a

### How did you install Ghost?

n/a

### Database type

MySQL 5.7

### Relevant log / error output

```shell

```

### Code of Conduct

- [x] I agree to be friendly and polite to people in this repository

**Comment by thibaultmol** [comment]
Forgot to link the lighthouse bug report I made: https://github.com/GoogleChrome/lighthouse/issues/16346

**Comment by kevinansfield** [comment]
`alt=""` is valid and describes the image as decorative meaning it won't cause screenreaders to repeat unhelpful image filenames. It doesn't sound like there's a bug in Lighthouse here?

Outputting `alt=""` for images in Ghost when no alt text is provided was an intentional choice due to Ghost's typical use-cases, where images will generally not be used to convey additional information that isn't already available in the caption or post body. When someone does want to add some additional context for an image outside of the caption/post content then they are free to add some alt text.

If you have some first-hand experience as to why that is unhelpful for assistive tech users then please let us know.

**Comment by kevinansfield** [comment]
Closing for now as this is the expected behaviour. If a strong argument can be made for why this is wrong and hurts the experience for the readers of Ghost sites then we can always re-open.
